src/app/components/countriesCard/Card.tsx

```diff
@@ -14,17 +14,17 @@ export default function CountryCard(props: CountryCardProps) {
   return (
     <section>
       <AppContainer>
-        <div className="w-80  h-custom-height shadow-lg rounded-xl bg-white cursor-pointer">
-        <div className="relative h-40 w-full shadow-md">
+        <div className="w-80 shadow-lg rounded-xl bg-primary-foreground cursor-pointer">
+          <div className="relative h-44 rounded">
             <Image
               src={props.flag}
               alt={props.alt}
               layout="fill"
-              objectFit="cover"
-              className="absolute inset-0 rounded-t-xl"
+              objectFit="contain"
+              className="inset-0 rounded-t-xl"
             />
           </div>
```

Comment:
Please review the logic here. Right now, your image is not taking the full width of the card